readme.md

```diff
@@ -140,6 +140,13 @@ Space around the box.
 
 Accepts a number or an object with any of the `top`, `right`, `bottom`, `left` properties. When a number is specified, the left/right margin is 3 times the top/bottom to make it look nice.
 
+##### backgroundColor
+
+Type: `string`<br>
+Values: `bgBlack` `bgRed` `bgGreen` `bgYellow` `bgBlue` `bgMagenta` `bgCyan` `bgWhite`
```

Comment:
The user shouldn't have to prefix the color with `bg` and camelCase. Just provide the colors and we can handle the rest in the code.